rider-fsharp/src/test/kotlin/com/jetbrains/rider/test/cases/FileSystemShimTest.kt

```diff
@@ -1,6 +1,8 @@
+package com.jetbrains.rider.test.cases
```

Comment:
Let's add some F#-specific bits to the package name, so it's easier to recognize it on CI?
We usually use something like `plugins.fsharp` after `rider` or `resharper` in similar cases.